api/src/identity-access-management/domain/usecases/authenticate-anonymous-user.usecase.js

```diff
@@ -19,15 +19,18 @@ export const authenticateAnonymousUser = async function ({
   audience,
   campaignToJoinRepository,
   userToCreateRepository,
+  anonymousUserTokenRepository,
   tokenService,
 }) {
   const campaign = await campaignToJoinRepository.getByCode({ code: campaignCode });
   if (!campaign.isSimplifiedAccess) {
     throw new UserCantBeCreatedError();
   }
 
-  const userToAdd = UserToCreate.createAnonymous({ lang });
-  const newUser = await userToCreateRepository.create({ user: userToAdd });
+  const user = UserToCreate.createAnonymous({ lang });
```

Comment:
Je n’ai jamais aimé le concept du model `UserToCreate`, mais à partir du moment où une instance de `UserToCreate` est utilisée, il est important de bien la distinguer et de ne pas la prendre pour une instance de `User`, d’où la suggestion suivante : 
```suggestion
  const userToCreate = UserToCreate.createAnonymous({ lang });
```